sjk-stacktrace/src/main/java/org/gridkit/jvmtool/stacktrace/StackFrame.java

```diff
@@ -216,6 +215,9 @@ public CharSequence subSequence(int start, int end) {
 
     @Override
     public int hashCode() {
+    	if(hash == 0) {
```

Comment:
I have strong opinion for hash in StackFrame to be final and ugliness of `hash = h = toString().hashCode();` is just one of the reasons.
I have changed hash code calculation not to use toString().